frontend/app/mission/new-mission.html

```diff
@@ -0,0 +1,158 @@
+<div class="wizard">
+    <a><span class="badge" ng->1</span> Basic Info</a>
+    <a><span class="badge">2</span> Activity Details</a>
+    <a class="current"><span class="badge badge-inverse">3</span> For Teachers</a>
+    <a><span class="badge">4</span> Review</a> 
+</div>
+
+<h1> Create a New Quest </h1>
+<p> Please fill out the requested information to author a new quest. </p>
+
+    <div class="container">
+      <h3>Here's Your Chance to Contribute</h3>
+
+  <div class="row">
+    <div class="col-sm-5" >
+            <label>Author's First Name</label>
+            <input class="form-control" type="text"   ng-model="users.name" placeholder="Enter Name">
+    </div>
+        <div class="col-sm-1" ></div>
+    <div class="col-sm-5" >       
+            <label>Company Name</label>
+            <input class="form-control " type="text"  ng-model="organization.name" placeholder="Enter Organization"></div>
+
+  </div>
+  <br>
+  
+  <div class="row">
+    <div class="col-sm-5" >
+            <label> Quest Title</label>
+            <input class="form-control" type="text" ng-model="quest.name" placeholder="Enter Quest Title">
+    </div>
+        <div class="col-sm-1" ></div>
+
+ <!-- <div class="col-sm-5" >       
+            <label> Quest Summary</label>
+         <textarea cols="40" rows="2" type="textarea"  ng-model="quest.summary" placeholder="Enter Summary"></div> 
+    </div> -->
+
+     <div class="row">
+    <div class="col-sm-12" >
+      <br>
+            <label> Keywords</label>
+            <input class="form-control" type="text" ng-model="quest_tags.name" placeholder="Enter Quest Keywords">
+    </div> 
+
+  </div>
+   <div style="width:80%; padding:10px; margin-left:15px;">
+  <div class="row">
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="3" ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/chemistry.png"></div>
```

Comment:
This seems a bit flat-footed.  Can we populate these things with a template loop rather than copy-pasting?